### PR TITLE
edits, actions, 404?

### PR DESCRIPTION
**Pull request by hougasian**

### CODE REVIEW

File: content/docs/concepts/errors.md [hunk 1]
```diff
@@ -3,61 +3,62 @@ title: Errors
...
-<div class="pill pill--circle-empty positive"></div>
-<div class="pill pill--circle-empty negative"></div>
-<div class="pill pill--circle-empty neutral"></div>
-<div class="pill pill--circle-empty-dot positive"></div>
-<div class="pill pill--circle-empty-dot negative"></div>
-<div class="pill pill--circle-empty-dot neutral"></div>
-<br><br>
-<div class="pill positive">
-  Positive
+<div  class="pill pill--circle-empty positive"></div>
```
Comment (tedk13) [inline content/docs/concepts/errors.md:24]: These extra spaces seem weird. Is that like a Markdown thing or something?

**Review by tedk13 — COMMENTED** [review]
Skypack sounds sweet

**Review by mrtrimble — APPROVED** [review]

**Review by jaime-lynn — APPROVED** [review]

File: content/docs/concepts/errors.md (hunk 1 above)
Comment (hougasian) [reply content/docs/concepts/errors.md:24]: yes, they do.

Comment (hougasian) [reply content/docs/concepts/errors.md]: Screen Shot 2022-04-11 at 1 17 01 PM

**Review by tedk13 — APPROVED** [review]
I feel better.

**Pull request merged**